.github/CODEOWNERS

```diff
@@ -2,4 +2,4 @@
 # This file controls who is tagged for review for any given pull request.
 
 # For anything not explicitly taken by someone else:
-* @eddynaka @michaelcfanning @marmegh
+*       @microsoft/1es-sdl-engineering @AllDwarf @martin-reznik @mkacmar
```

Comment:
Would you mind referencing just the team instead of specific usernames? I think our future selves will thank us if we ever need to deal with offboarding.